tests/test_config.py

```diff
@@ -45,7 +45,6 @@ def _assert_defaults(self, cfg: config.Config):
         assert cfg.chat.context == "default"
         assert cfg.chat.session is None
         assert cfg.chat.logs_dir == f"{data_dir}/chatlogs"
-        assert not cfg.chat.greedy_mode
```

Comment:
Do we need to add a check here for temperature?

Reply:
This is done as part of the latest patchset